### PR TITLE
[Quantization] Quantization docs updates

### PR DESCRIPTION
**Pull request by beicy**
*Description*:
1. Added "loading quantized caffe2 models"
2. Added one of our previous quantization design: forcing some types of quantized nodes output have the same quantization param as the input 

*Testing*:
N/A
*Documentation*:
[Optional Fixes #issue]

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

File: docs/Quantization.md [hunk 1]
```diff
@@ -115,6 +129,35 @@ the quantized text translator:
...
+```
+- Supported int32 quantized bias.
+
+In most of the cases, bias is quantized in int32 for the precision purpose (the partial sum of the matrix-matrix multiplication is accumulated into int32, so int32 bias can be added to the int32 partial sum for better accuracy). Glow now supports int32 quantized bias in ```Convolution```, ```FullyConnected``` and ```RowwiseQuantizedFullyConnected``` nodes.
+
+- Supported the conversion from uint8 quantized activations to int8 quantized activations.
+
+For the quantized Caffe2 ops, the activations are quantized to uint_8.
+In Glow, the activations are quantized to int_8. Therefore, for the offset read from quantized 
+caffe2 model, we need to subtract 128(i.e. INT8_MIN) to make the activations becomes int8_t.
```
Comment (bertmaher) [inline docs/Quantization.md]: grammar: "becomes" -> "become"

File: docs/Quantization.md [hunk 2]
```diff
@@ -57,7 +57,21 @@ inference. Then, we recompile the network using this profile information to
...
 computation and aim to generate outputs in the range that the original
-floating-point network produces. The figure below shows a quantized subgraph
+floating-point network produces. During the conversion, for the following types of 
+quantized nodes, we ignore the output's quantization params(if they are provided) and 
+force the output have the same quantization params as the input for performace purpose:
+```
+LocalResponseNormalizationNode                       
+SigmoidNode                                         
+SliceNode                                      
+ReshapeNode
```
[the remark this answers is not in the document]
Comment (beicy) [reply docs/Quantization.md]: Yes, I agree. In our profiling, the quantization parameters of ReshapeNode input and output should be the same and there is no need to force it. But I think it should be OK to put ReshapeNode here just for emphasizing ?

File: docs/Quantization.md [hunk 3]
```diff
@@ -115,6 +129,35 @@ the quantized text translator:
...
+Int8MaxPool
+Int8AveragePool
+Int8FC
+Int8SumRelu
+Int8GivenIntTensorFill
+Int8GivenTensorFill
+```
+- Supported int32 quantized bias.
+
+In most of the cases, bias is quantized in int32 for the precision purpose (the partial sum of the matrix-matrix multiplication is accumulated into int32, so int32 bias can be added to the int32 partial sum for better accuracy). Glow now supports int32 quantized bias in ```Convolution```, ```FullyConnected``` and ```RowwiseQuantizedFullyConnected``` nodes.
```
[the remark this answers is not in the document]
Comment (beicy) [reply docs/Quantization.md]: For <Gemm> op in ONNX, we don't support "int32" bias so far. When we load `Gemm` op, we decompose it into `MatMul` and `Add` nodes.  So in our quantization, there is no quantization params for <result of Gemm>.   If you mean the scale for result of `MatMul` and the result `Add` should be the same, I think this should be left as backend decision.

[the remark this answers is not in the document]
Comment (beicy) [reply docs/Quantization.md]: For op  `Gemm`,  the quantized subgraph should be: 
```
     MatMul (quantized)         bias
             |                    |
             |           QuantizeNode
             \               /
                Add (quantized)
```
If we want a simple "add" without considering scaling, the scales of MatMul node and QuantizeNode in the above graph should be the same.  However, the range of `bias ` is independent from the result of MatMul, I don't think we could force their scales to be the same.

However, for quantized Add, we could check if the operands have the same quantization params with the result, and if so, we can simply do add.

File: docs/Quantization.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rdzhabarov) [reply docs/Quantization.md]: let's keep it for completeness

**Pull request merged**